content/user-guide/raspberry.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Raspberry guide
+description: How to run Mysterium dVPN on Raspberry Pi 
+---
+
+## Spin-off Raspberry Pi node in consumer mode
+
+1.  SSH into your RaspberryPi and open editor of /lib/systemd/system/mysterium-node.service file.
+2.  Replace the following line:
+
+```bash
+ExecStart=/usr/bin/myst $CONF_DIR $SCRIPT_DIR $DATA_DIR $RUN_DIR $DAEMON_OPTS service --agreed-terms-and-conditions $SERVICE_OPTS`
+```
+with:
+
+```bash
+ExecStart=/usr/bin/myst $CONF_DIR $SCRIPT_DIR $DATA_DIR $RUN_DIR $DAEMON_OPTS daemon
+```
+
+3. To apply the changed configuration you will need to reboot host or re-read configuration and restart it via terminal:
+
+`sudo systemctl daemon-reload`
+
+`sudo systemctl restart mysterium-node`
+
+After thats done follow our linux guide on `Basic` or `Advanced` consumer node operation using our provided `cli` applications.
```

Comment:
If anyone knows how I could point to the linux docs file here, let me know please.